pages/home/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { alpha, Box, Button, Typography, useTheme } from "@mui/material";
+import Grid from "@mui/material/Unstable_Grid2";
+import Image from "next/image";
+import Card from "../../components/common/Card";
+import Layout from "../../components/common/Layout";
+import { tokens } from "../../constants/color-palette";
+import sketchPhoto from "../../assets/sketchPhoto.png";
+
+export default function Home() {
+  const theme = useTheme();
+  const colors = tokens(theme.palette.mode);
+
+  const fetchFromCookies = "Łukasz";
+
+  return (
+    <Layout page={"Home"}>
+      <Grid xs={4}>
+        <Card bgcolor={alpha(theme.palette.secondary.main, 0.2)}>
+          <Box width="100%" display="flex" flexDirection="column">
+            <Typography variant="h4" fontWeight="700" letterSpacing="1px">
+              Witaj z powrotem,
+            </Typography>
+            <Typography variant="h4" fontWeight="700" letterSpacing="1px">
+              {fetchFromCookies}!
+            </Typography>
+          </Box>
+          <Box marginTop="1.5rem" width="100%">
+            <Typography width="85%" variant="h4" fontWeight="500" align="justify">
+              Jeśli potrzebujesz wsparcia w obsłudze systemu, skorzystaj z naszego szybkiego poradnika. Życzymy owocnej
+              pracy!
+            </Typography>
+          </Box>
+          <Box marginTop="1.5rem" width="100%">
+            <Button
+              variant="contained"
+              color="secondary"
+              sx={{
+                borderRadius: "0.5rem",
```

Comment:
borderRadius 0.6rem looks more like design in figma

Reply:
good point, done